substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -418,7 +418,7 @@ def native_unittests_task():
     additional_build_args = [
         '-H:AdditionalSecurityProviders=com.oracle.svm.test.SecurityServiceTest$NoOpProvider',
         '-H:AdditionalSecurityServiceTypes=com.oracle.svm.test.SecurityServiceTest$JCACompliantNoOpService',
-        '-H:+AllowVMInspection'
+        '-H:+AllowVMInspection',
```

Comment:
Leftover from reverting to put `--initialize-at-build-time=...` here.

Reply:
Sorry for my late response. I made requested changes, and everything should work now.